modules/microshift-get-rpm-release-info.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+//microshift_updating/microshift-list-update-contents.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-get-rpm-release-info_{context}"]
+= List the contents of the {microshift-short} RPM release package
+
```

Comment:
Should a step be included to see what versions of the microshift-release RPM is available for download ? 

sudo dnf repoquery microshift-release-info-0:4.16.*